### DISCUSSION TITLE
Can it support. net8

### DISCUSSION THREAD

**Discussion by dfengpo** [rabbitmq/rabbitmq-dotnet-client#1658] [General]
### Is your feature request related to a problem? Please describe.

Can it support. net8

### Describe the solution you'd like

Can it support. net8

### Describe alternatives you've considered

Can it support. net8

### Additional context

Can it support. net8

**Comment by michaelklishin** [comment]
@dfengpo either provide details (e.g. what specifically you have tried to do and what were the errors on .NET 8) or contribute improvements, or post five word-strong problem definitions elsewhere.

Or just give 7.0 previews a shot on NET 8.

**Comment by lukebakken** [comment]
This information is available on nuget.org:

https://www.nuget.org/packages/RabbitMQ.Client/#supportedframeworks-body-tab
